examples/chip-tool/templates/commands.zapt

```diff
@@ -186,9 +186,9 @@ public:
     {
         {{#chip_server_cluster_command_arguments}}
         {{#if (isString type)}}
-        AddArgument("{{asCamelCased label}}", &m{{asCamelCased label false}});
+        AddArgument("{{asCamelCased label false}}", &m{{asCamelCased label false}});
```

Comment:
For future PRs we may want to look into API readability: the false argument is not obvious in purpose. Maybe something like `startUpperCamelCase` and `startLowerCamelCase` methods would be more readable, although I find those namings awkward as well. Can these be chained? like `camelCase label | uppercaseFirstLetter`